README.md

```diff
@@ -90,6 +90,7 @@ The EBS CSI Driver is compatible with Kubernetes versions v1.17+ and implements
 * [Driver Installation](docs/install.md)
 * [Driver Launch Options](docs/options.md)
 * [StorageClass Parameters](docs/parameters.md)
+* [Troubleshooting and frequently asked questions](docs/faq.md)
```

Comment:
**Astronomical** np: `frequently asked questions` -> `Frequently Asked Questions` for consistency with the surrounding text.

Reply:
additional np: Remove "troubleshooting", as the doc is purely an FAQ